index.js

```diff
@@ -27,6 +27,7 @@ var Parser = function (opts) {
 
   this.headers = opts.headers || null
   this.strict = opts.strict || null
+  this.transformHeaders = opts.transformHeaders || function (id) { return id }
```

Comment:
I'd prefer the name mapHeaders since this is like array.map. Minor detail, could you move `function (id) { return id }` outside of the constructor and name it? fx `defaultMapHeaders` or something